jerry-core/vm/opcodes-ecma-equality.c

```diff
@@ -44,6 +44,18 @@ ecma_value_t
 opfunc_equal_value (ecma_value_t left_value, /**< left value */
                     ecma_value_t right_value) /**< right value */
 {
+  JERRY_ASSERT (!ecma_is_value_error (left_value)
+                && !ecma_is_value_error (right_value));
+
+  if (ecma_are_values_integer_numbers (left_value, right_value))
+  {
+    if (left_value == right_value)
+    {
+      return ecma_make_simple_value (ECMA_SIMPLE_VALUE_TRUE);
+    }
+    return ecma_make_simple_value (ECMA_SIMPLE_VALUE_FALSE);
```

Comment:
But I would need to use it everywhere in this file for consistency, and (ecma_integer_value_t) left_value <= (ecma_integer_value_t) right_value is too long for checks.